projects/core/src/lib/map/map-layer.component.ts

```diff
@@ -311,12 +320,16 @@ export class OlRasterLayerComponent
 
         this.source.setTileLoadFunction((tile, src) => {
             const client = new XMLHttpRequest();
-
-            const sub = this.projectService.createQueryAbortStream().subscribe(() => {
-                (tile as any).getImage().src = 'assets/images/empty_tile.png';
-                client.abort();
-                sub.unsubscribe();
-                this.source.dispatchEvent('tileloaderror');
+            const tileZoomLevel: number = (tile as any).getTileCoord()[0];
+            const tileResolution = this.mapService.getView().getResolutionForZoom(tileZoomLevel);
+
+            const sub = this.projectService.createQueryAbortStream().subscribe((newResolution) => {
+                if (newResolution !== tileResolution) {
```

Comment:
Does this mean we do not abort now when the user pans?

We should abort there, too.

Reply:
d3de124e84e0164decaf638cfe0aa245d8068a25